src/store/rootReducer.ts

```diff
@@ -17,6 +18,7 @@ const rootReducer = combineReducers({
   hotspotDetails: hotspotDetailsSlice.reducer,
   hotspots: hotspotsSlice.reducer,
   hotspotChecklist: hotspotChecklistSlice.reducer,
+  networkHotspots: networkHotspotsSlice.reducer,
```

Comment:
Maybe we should cache this reducer using redux-persist or something?

Reply:
i was thinking about that, but figured for now it'd be easier to get out of a pickle by having someone force quit the app, in case something bad got stuck in there. we can revisit that if performance becomes an issue either in the app or the api